CHANGELOG.md

```diff
@@ -18,10 +18,10 @@
 
 ### 1.8.26
 
-
 - Deploy script syncs local data with Metaphysics - ash
 - Updates drawer sub-scrollview enabled state when tabs change - ash
 - Get rid of unneeded City render pass and potential further deep renders by unstable inline function props - alloy
+- Updates copy in Global Saves and Follows Shows tab - ashley
```

Comment:
`1.8.26` is already released, can we move this to the top of the changelog, under `### Master`?